producers-api/build.gradle

```diff
@@ -28,3 +28,73 @@ bootJar {
 test {
     useJUnitPlatform()
 }
+
+ext.cxfVersion = "3.5.4"
+ext.jaxwsVersion = "2.3.1"
+
+configurations {
+  wsdl2java
+}
+
+dependencies {
+    wsdl2java("javax.jws:javax.jws-api:1.1")
+    wsdl2java("javax.xml.ws:jaxws-api:${jaxwsVersion}")
+    wsdl2java("javax.xml.ws:jaxws-api:${jaxwsVersion}")
+    wsdl2java("org.apache.cxf:cxf-tools:${cxfVersion}")
+    wsdl2java("org.apache.cxf:cxf-tools-wsdlto-core:${cxfVersion}")
+    wsdl2java("org.apache.cxf:cxf-tools-wsdlto-frontend-jaxws:${cxfVersion}")
+    wsdl2java("org.apache.cxf:cxf-tools-wsdlto-databinding-jaxb:${cxfVersion}")
+    wsdl2java("org.apache.cxf:cxf-spring-boot-starter-jaxws:${cxfVersion}")
+    wsdl2java("org.apache.cxf:cxf-rt-transports-http-jetty:${cxfVersion}")
+    wsdl2java("org.apache.cxf:cxf-rt-features-logging:${cxfVersion}")
+}
+
+task sdtEndPointWsdl2Java(type: JavaExec) {
+  main = "org.apache.cxf.tools.wsdlto.WSDLToJava"
+  classpath = configurations.wsdl2java
+  args = [
+    "-d", "src-gen/main/java",
+    "-fe", "jaxws21",
+    "-b", "${projectDir}/${wsdlXsdResourcesDir}/jaxb-bindings.xml",
+    "${projectDir}/${wsdlXsdResourcesDir}/wsdl/SdtEndpoint.wsdl"
+  ]
+}
+
+task sdtInternalEndpointWsdl2Java(type: JavaExec) {
+  main = "org.apache.cxf.tools.wsdlto.WSDLToJava"
+  classpath = configurations.wsdl2java
+  args = [
+    "-d", "src-gen/main/java",
+    "-fe", "jaxws21",
+    "-b", "${projectDir}/${wsdlXsdResourcesDir}/jaxb-bindings.xml",
+    "${projectDir}/${wsdlXsdResourcesDir}/wsdl/SdtInternalEndpoint.wsdl"
+  ]
+}
+
+task targetAppInternalEndpointWsdl2Java(type: JavaExec) {
+  main = "org.apache.cxf.tools.wsdlto.WSDLToJava"
+  classpath = configurations.wsdl2java
+  args = [
+    "-d", "src-gen/main/java",
+    "-fe", "jaxws21",
+    "-b", "${projectDir}/${wsdlXsdResourcesDir}/jaxb-bindings.xml",
+    "${projectDir}/${wsdlXsdResourcesDir}/wsdl/TargetAppInternalEndpoint.wsdl"
+  ]
+}
+
+task sdtEndpointDistWsdl2Java(type: JavaExec) {
+  main = "org.apache.cxf.tools.wsdlto.WSDLToJava"
+  classpath = configurations.wsdl2java
+  args = [
+    "-d", "src-gen/main/java",
+    "-fe", "jaxws21",
+    "-b", "${projectDir}/${wsdlXsdResourcesDir}/jaxb-bindings.xml",
+    "${projectDir}/${wsdlXsdResourcesDir}/wsdl/SdtEndpointDist.wsdl"
+  ]
+}
```

Comment:
This task only applies to the sdt-consumer-sample module - see the generate-from-wsdl-client ant target in sdt-build/build.xml which is only called for sdt-consumer-sample.  This task, plus the associated reference below, needs to be removed from here. 
 It will need to be added and added to the sdt-consumer-sample module.

Reply:
Confirmed that task has been removed.  Task still needs to be added to sdt-consumer-sample module though.